src/Compute/custom/Strategies/ComputeRp/ModelAdapters/AvailabilitySet.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Microsoft.Azure.Commands.Compute.Strategies.ComputeRp;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.PowerShell.Cmdlets.Compute.Models.Api20190301
+{
+    public partial class AvailabilitySet : IResource
```

Comment:
@markcowl any reason that `IResource` doesn't need the full namespace here but the other classes down below have the full namespace?